mifosng-android/src/main/java/com/mifos/mifosxdroid/adapters/CentersListAdapter.java

```diff
@@ -46,21 +55,35 @@ public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(ViewHolder holder, int position) {
 
-        String centerId = context.getResources()
-                .getString(R.string.center_id) + centers.get(position).getId();
+        centerId = "" + centers.get(position).getId();
```

Comment:
It should be like this.
```java
@Override
    public void onBindViewHolder(ViewHolder holder, int position) {
        Center center = centers.get(position);
 
        holder.tvAccountNumber.setText("Account No: " + String.valueOf(center.getAccountNo()));
        holder.tvCenterId.setText(String.valueOf(center.getId()));
        holder.tv_center_name.setText(center.getName());
        if (center.getStaffId() != null) {
           holder.tvStaffId.setText(String.valueOf(center.getStaffId()));
           holder.tvStaffName.setText(center.getStaffName());
        } else {
            staffId = "";
            holder.tvStaffName.setText("no staff");
        }
        holder.tvOfficeId.setText(String.valueOf(center.getOfficeId()));
        holder.tvOfficeName.setText(center.getOfficeName());
        if (center.getActive()) {
            holder.iv_status_indicator.setImageDrawable(
                    Utils.setCircularBackground(R.color.light_green, context));
        } else {
            holder.iv_status_indicator.setImageDrawable(
                    Utils.setCircularBackground(R.color.light_red, context));
        }
    }
```
**Note** Please include Hard Coded String in strings.xml